src/Monoid/StringMonoid.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Widmogrod\Monoid;
+
+use Widmogrod\FantasyLand;
+
+class StringMonoid implements FantasyLand\Monoid
```

Comment:
Please reuse/replace this with recently added structure `Widmogrod\Primitive\Stringg`